test/integration/Delegation.js

```diff
@@ -219,8 +219,10 @@ describe("Delegation", () => {
         ).to.be.reverted
     })
 
-    it("delegator 1 partially unbonds twice (at different times), rebonds with one unbonding lock and withdraws with the other", async () => {
+    it("delegator 1 partially unbonds twice (at different times), rebonds with one unbonding lock and withdraws with the other AFTER roundLength update", async () => {
```

Comment:
It seems that there are two separate test scenarios here:

1. The scenario for an unbond, rebond, withdraw *without* a roundLength update
2. The scenario for an unbond, rebond, withdraw *with* a roundLength update

These two scenarios are very similar and are only distinguished by the existence of a roundLength update. But, for test structure clarity I think it would be beneficial to have a clear distinction between the two scenarios because we want clear confirmation that the unbond, rebond, withdraw flow works properly regardless of whether or not a roundLength update occurred.

I think an easy to way to achieve this clarity would be to move the logic in this block into a helper function that is executed once for scenario 1 and once for scenario 2. And for scenario 2 we would just execute the roundLength update before executing the helper.